src/assets/styles/custom.scss

```diff
@@ -0,0 +1,10 @@
+$primary: #8b65ad;
+$secondary: #6e3f98;
+$dark: #42265b;
+
+.card {
+  box-shadow: 0 0 15px 0 #000;
+  background: linear-gradient(180deg, #855798 6%, #cca097 75%);
+  background-color: #42265b00 !important;
```

Comment:
Using !important is bad practice and should be avoided. I think you can create a new CSS class for this background.